src/field/vector3d.cxx

```diff
@@ -160,6 +163,8 @@ Vector3D & Vector3D::operator=(const Vector3D &rhs) {
 }
 
 Vector3D & Vector3D::operator=(const Vector2D &rhs) {
+  fielddatamesh = rhs.x.getMesh();
```

Comment:
Should this be `getDataMesh()` for consistency? (I know they should return the same thing)